draft-irtf-cfrg-rsa-blind-signatures.md

```diff
@@ -346,12 +473,12 @@ The RSASSA-PSS parameters, defined as in {{!RFC8017, Section 9.1.1}}, are as fol
 - MGF: mask generation function
 - sLen: intended length in octets of the salt
 
-It is RECOMMENDED that implementations support the following encoding options:
+Implementations that expose the interface in {{salted-interface}} are RECOMMENDED to
```

Comment:
Now that the paper is up, we should link to it and use it as justification for the additional salting

Reply:
Yep, good point. I'll add a reference.

Reply:
Added!